CHANGELOG.md

```diff
@@ -1,9 +1,7 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+-   VoterListItem component added
```

Comment:
You can keep the current text on top ("All notable changes..") and move these updates under the `## [Unreleased]` header:
```
# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

### Added

-   New `VoterListItem` Component

### Changed

-   `VotersTable` component according to new ODS designs

## [0.2.10] - 2023-08-08

[...]
```